src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -1,21 +1,18 @@
 import React, { useCallback, useEffect } from "react";
-import { Button, Form } from "react-bootstrap";
+import { Form } from "react-bootstrap";
 import { useForm } from "react-hook-form";
-// Hooks
```

Comment:
Removed import comments as to not trip over the automated import sorting algorithm.